source/javascripts/components/ProductTour/Highlighter.scss

```diff
@@ -0,0 +1,30 @@
+.highlighter {
```

Comment:
The styles look good here, but shouldn't this be placed in BitKit? I thought that was the approach to take with new styles/UI components.

Reply:
The bitkit version is pre chakra based. Mate already contacted the platform team if they could upgrade and migrate the components to the latest bitkit version.

When the upgrade happens it will simplify a lot of things, like using the included popover instead of manually creating it and positioning it like in here. I don't know when exactly they can pick up this task, so I went ahead with it this way. 

I just didn't want to move everything into style props and that's why there are css files. My thought process was that these will be refactored when the chakra powered bitkit will arrive.